ONSdigital/eq-questionnaire-schemas
# Add unrelated block to relationships

### What is the context of this PR?

Adds the "Related to anyone else?" block to relationships.

### How to review

- Check out the https://github.com/ONSdigital/eq-questionnaire-runner/pull/379 runner change
- Build and link the schemas into runner
- Follow the steps in the feature https://github.com/ONSdigital/eq-questionnaire-runner/pull/368 to test the schemas

### Checklist

- [ ] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

### Quick Launch

**The quick launch schemas won't work until the linked runner pr is merged**

#### England

[Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=H&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/add-unrelated-block-to-relationships/schemas/en/census_household_gb_eng.json)

#### Northern Ireland

[Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/add-unrelated-block-to-relationships/schemas/en/census_household_gb_nir.json)
